data_steward/retraction/retract_deactivated_pids.py

```diff
@@ -0,0 +1,537 @@
+# Python imports
```

Comment:
I ran pylint here and got these relevant suggestions.
```
 pylint data_steward/retraction/retract_deactivated_pids.py 
************* Module data_steward.retraction.retract_deactivated_pids
data_steward/retraction/retract_deactivated_pids.py:177:8: W0612: Unused variable 'i' (unused-variable)
data_steward/retraction/retract_deactivated_pids.py:238:12: W0612: Unused variable 'i' (unused-variable)
data_steward/retraction/retract_deactivated_pids.py:17:0: C0412: Imports from package constants are not grouped (ungrouped-imports)
```
When you don't really care about the output you can use an underscore instead, `_`.

Reply:
@amcgrenera-vumc Please group related imports.
```
pylint data_steward/retraction/retract_deactivated_pids.py 
************* Module data_steward.retraction.retract_deactivated_pids
data_steward/retraction/retract_deactivated_pids.py:17:0: C0412: Imports from package constants are not grouped (ungrouped-imports)
```